app/main/forms/auth_forms.py

```diff
@@ -9,7 +10,9 @@ class LoginForm(Form):
         'Email address', id="input_email_address",
         validators=[
             DataRequired(message="You must provide an email address"),
-            Email(message="You must provide a valid email address")
+            Regexp("^[^@^\s]+@[^@^\.^\s]+(\.[^@^\.^\s]+)+$",
+                   flags=re.IGNORECASE,
```

Comment:
Don't think we need this flag now, as the regex doesn't specify any "cased" characters, just `@` and `.`.